tinkerpop3/src/test/java/io/shiftleft/overflowdb/GraphBuilderTest.java

```diff
@@ -0,0 +1,37 @@
+package io.shiftleft.overflowdb;
+
+import io.shiftleft.overflowdb.testdomains.simple.SimpleDomain;
+import io.shiftleft.overflowdb.testdomains.simple.TestEdge;
+import io.shiftleft.overflowdb.testdomains.simple.TestNode;
+import org.apache.tinkerpop.gremlin.structure.Direction;
+import org.apache.tinkerpop.gremlin.structure.T;
+import org.junit.Assert;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import java.util.Arrays;
+
+public class GraphBuilderTest {
+  @Test
+  @Ignore
```

Comment:
drop?

Reply:
I've enabled this test, it is just a sanity check for nodes, properties in the graph builder.

Reply:
good, just to clarify my comment 'drop?' was referring to just that one `@Ignore`, not the entire test :)